### ISSUE TITLE
about orderbook

### ISSUE THREAD

**Issue by fm612** [satoshipay/solar#978]
solar wallet can look orderbook   like  stellarterm or blockeq  is best  .
only can see price now, canot look hou many can buy(or sell) in a price

**Comment by andywer** [comment]
Yeah, that's true – being able to look into the orderbook would be nice. Cannot tell you when we will be able to work on that, though, as we are not that trading-focussed.

Will probably take a while until we find time to work on this – we already have a huge backlog of things we need to work on.
